### ISSUE TITLE
Location edits don't account for relative paths

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#521]
[This discourse issue](https://discourse.osc.edu/t/rstudio-help-link-error/933) encountered an issue with our Location Header edits (for redirects) that don't correctly account for relative URLs like `../../foo`. 

Here are details on what it is and what it should be.  This can be replicated by opening Rstudio and attempting to click on any of the help links embedded in the initial help document for a given package.
```text
# the link reference location
../../igraph/html/make_ring.html

# the initial 302 redirection. Looks like we inserted the /rnode/<host>/<port> incorrectly
../rnode/o0519.ten.osc.edu/23146/../igraph/html/make_ring.html

# what it ends up being
/rnode/o0519.ten.osc.edu/23146/help/library/igraph/rnode/o0519.ten.osc.edu/igraph/html/make_ring.html

# what it should be, inserted /rnode/<host>/<port> in the front, not both in the front and in the middle.
 /rnode/o0519.ten.osc.edu/23146/help/library/igraph/html/make_ring.html
```

**Comment by msquee** [comment]
See #493

**Comment by johrstrom** [comment]
This was fixed in #1105.
